tools/board.py

```
@@ -371,6 +444,37 @@ def open_issues() -> dict[int, str]:
+
+    A parent that is closed, or that carries no CAUSE_LABEL, yields nothing --
+    the first because a closed cause blocks nothing, the second because an
+    unlabelled link is task decomposition and means something else entirely.
+    """
+    parents: dict[int, int] = {}
+    cursor = "null"
+    while True:
+        data = gql(
+            "query($o:String!,$r:String!){repository(owner:$o,name:$r){"
+            "issues(states:OPEN,first:100,after:%s){pageInfo{hasNextPage endCursor}"
+            "nodes{number parent{number state labels(first:50){nodes{name}}}}}}}" % cursor,
```

Ruff (0.16.3)

[warning] Use format specifiers instead of percent format

Replace with format specifiers

(UP031)